assets/js/view/auction_round.rb

```diff
@@ -20,7 +20,6 @@ def render
       h(:div, [
         h(UndoAndPass, undo: @game.actions.size.positive?),
         *render_companies,
-        render_input,
         h(View::Players, game: @game),
       ].compact)
     end
```

Comment:
clean up action_round, remov selected_company, actionable and imports as they aren't used